lib/ansible/modules/system/iptables_raw.py

```diff
@@ -0,0 +1,556 @@
+    version_added: "2.7"
+  chains_present:
+    description:
+      - List of chains to be present.
+    required: false
+    default: null
+    version_added: "2.7"
+  chains_absent:
+    description:
+      - List of chains to be absent.
+    required: false
+    default: null
+    version_added: "2.7"
+  chains_policy:
+    description:
+      - List of chains and policy, separated by space in that order, to enforce.
+    required: false
+    default: null
+    version_added: "2.7"
+  chains_rename:
+    description:
+      - List of source chains and destination chains, separated by space in that
+        order, to rename. If source is not present but destination is, it is
+        assumed the operation has been previously run and will NOT set change or
+        failure. If neither source or destination are found, failure is set.
+        Similarly, failure is also set if both are present at the same time.
+    required: false
+    default: null
+    version_added: "2.7"
+  chains_flush:
+    description:
+      - List of chains to flush. When there are no underlying rules, nothing
+        will be done and therefore change will not be set.
+    required: false
+    default: null
+    version_added: "2.7"
+  rules_append:
+    description:
+      - List of rules to append into the chain, if not present. This should
+        look exactly as you would type out via iptables cli, without the
+        preceding '-A '.
+    required: false
+    default: null
+    version_added: "2.7"
+  rules_insert:
+    description:
+      - List of rules to insert into the chain, if not present. This should
+        look exactly as you would type out via iptables cli, without the
+        preceding '-I '.
+    required: false
+    default: null
+    version_added: "2.7"
+  rules_absent:
+    description:
+      - List of rules that should be absent from the chain.
+    required: false
+    default: null
+    version_added: "2.7"
+"""
+
+EXAMPLES = """
+# Allow port 22 tcp (SSH) from anywhere, appended at the bottom of the chain.
+- iptables_raw:
+    ipv: 4
+    table: filter
+    rules_append:
+      - { 1: "INPUT -p tcp -m state --state NEW -m tcp --dport 22 -j ACCEPT" }
+
+# Remove SSH rule
+- iptables_raw:
+    rules_absent:
+      - { 1: "INPUT -p tcp -m state --state NEW -m tcp --dport 22 -j ACCEPT" }
+
+# The following will do, in order:
+# 1) Add chain 'WWW_SERVICES'.
+# 2) Append -j DROP to WWW_SERVICES (at the very bottom).
+# 3) Insert tcp port 80 rule to WWW_SERVICES (above the DROP rule).
+# 4) Insert tcp port 443 rule to WWW_SERVICES (above the port 80 rule).
+# 5) Set default policy for INPUT to 'ACCEPT'.
+# 6) Append -j DROP to INPUT.
+# 7) Insert tcp port 22 rule to INPUT.
+# 8) Remove tcp port 21 rule from INPUT.
+# 9) Flush the 'TESTING' table.
+# 10) Remove 'OLD_CHAIN' chain.
+# 11) Rename chain 'SRC_CHAIN' to 'DST_CHAIN'.
+# 12) Append -j MASQUERADE for eth0 on the POSTROUTING chain on the nat table.
+
+- iptables_raw:
```

Comment:
Please set a `name:` for all EXAMPLES
```suggestion
- name: Example
  iptables_raw:
```